tests/test_integration/test_suites.py

```diff
@@ -617,3 +617,69 @@ def definition(outfield, cond):
 
     def validation(outfield, cond, *, domain, origin, **kwargs):
         outfield[np.array(cond, dtype=np.bool_)] = 1
+
+
+class TestLowDimensionalFields(gt_testing.StencilTestSuite):
+    dtypes = {"field_in": np.float64, "another_field": np.float64, "field_out": np.float64}
+    domain_range = [(4, 10), (4, 10), (4, 10)]
+    backends = ["debug", "numpy", "gtc:gt:cpu_ifirst", "gtc:gt:cpu_kfirst"]
+    symbols = {
+        "field_in": gt_testing.field(
+            in_range=(-10, 10), axes="K", boundary=[(0, 0), (0, 0), (0, 0)]
+        ),
+        "another_field": gt_testing.field(
+            in_range=(-10, 10), axes="IJ", boundary=[(1, 1), (1, 1), (0, 0)]
+        ),
+        "field_out": gt_testing.field(
+            in_range=(-10, 10), axes="IJK", boundary=[(0, 0), (0, 0), (0, 0)]
+        ),
+    }
```

Comment:
👍  This is excellent. Thanks for adding this to the test suites. I had wanted to do that originally, but wasn't familiar enough with how it worked at the time.

Reply:
It wasn't easy, suites code is ugly and quite old...